layouts/_default/baseof.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="{{ .Site.Language.Lang }}">
   {{- partial "head.html" . -}}
-  <body dir="ltr">
+  <body dir="{{ .Site.Language.LanguageDirection }}">
```

Comment:
```suggestion
  <body dir="{{ .Site.Language.LanguageDirection | default `ltr` }}">
```

I think we need to default to `ltr` for backward compatibility

Reply:
Ok, Updated the PR.